### DISCUSSION TITLE
Valheim PLUS

### DISCUSSION THREAD

**Discussion by ElZeRo69** [GameServerManagers/LinuxGSM#3897] [Q&A] [answered]
I have installed valheim plus but can not get the server function to work.
My settings in the config:
[Server]
enabled=true
maxPlayers=20
disableServerPassword=true
enforceMod=true
serverSyncsConfig=true
serverSyncHotkeys=false

In the server list ingame its shown with MaxPlayers = 10 and Password on

I have other settings in PLUS mode that are working ,, So i know that valheim PLUS is correct installed.
The only problem is the "server" section. 

Does someone have an solution ?

**Comment by ijyt** [comment, marked as answer]
The disableServerPassword option for V+ is only for local servers, rather than dedicated ones unfortunately, likely this will also be true of the maxPlayers option.
